src/atoms/FlashMessage/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import styles from './flash_message.module.scss';
+
+function FlashMessage(props) {
+  return (
+    <div className={classnames(styles['flash-message'], styles[props.variant])}>
```

Comment:
@patrickkim Why did you break away from our convention of deconstructing the `props` here? I know it's only two, but it's nicer if someone wants to add a third down the line.